### PR TITLE
Project completed and apply for checkup

### PR DESCRIPTION
**Pull request by GzimAsani**
We have completed the project and its requirements and now we are submitting it for a review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,58 @@
-# Microverse-first-html-project
\ No newline at end of file
+# Microverse-first-html-project
+First project of Rayan Rassam and Gzim Asani.
+
+Additional description about the project and its features.
+
+This collaborative project displays our knowledge of HTML and CSS. In this project we have recreated the basic interface of Youtube.
+
+<img src='./media/screenshot.png'>
```
Comment (krys2fa) [inline README.md]: - [x] [REQUIRED] Please change this image to a more suitable one. The texts in the side bar are mashed together in the image. It does not make your README look professional.

File: index.html [hunk 2]
```diff
@@ -1,11 +1,407 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
```
Comment (krys2fa) [inline index.html]: - [x] [REQUIRED] There is no need to have the meta tags and title taken out. They help search engines understand your page. Please put them back and replace the title with a more suitable one. Eg: Youtube Clone.

File: index.html [hunk 3]
```diff
@@ -1,11 +1,407 @@
...
+            <div class="commentor-input-container">
+               <div class="inline-block">
+                  <h4 class="inline-block">Jackson Lee</h4>
+                  <p class="inline-block">2 years ago</p>
+               </div>
+               <p>flujo completo de desarrollo desde una sola herramienta a diferencia de GitHub, aprende a utilizarla desde la planificación del proyecto, el manejo</p>
+               <div class="margin-top: 0;">
+                 <p style="display: inline-block;">&#128077; 500</p>
+                 <p style="display: inline-block;">&#128078; 12 </p>
+                 <p style="display: inline-block;">REPLY</p>
```
Comment (krys2fa) [inline index.html]: - [x] [REQUIRED] Inline styling are frowned upon. They are considered [bad CSS practice](https://speckyboy.com/good-bad-css-practices). Please move all inline styles in this and all other sections to an external stylesheet.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+.iconu {
+  color:red;
+  transform: scale(4.0);
+}
+.icon {
+  transform: scale(2.0);
+}
+.icon {
+  transform: (1);
+}
```
Comment (krys2fa) [inline style.css]: - [x] [REQUIRED] Kindly remove the duplicate selector for this rule.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+
+.horizontal-line{
+  color: grey;
+  width: 74%;
+  margin-left: 0;
+}
+
+.channel-details-container{
+
+}
```
Comment (krys2fa) [inline style.css]: - [x] [REQUIRED] This rule is empty and not necessary. Kindly remove this.

**Review by krys2fa — CHANGES_REQUESTED** [review]
## Awesome work @GzimAsani and @Rayan84 👏 
Your project displays good use of CSS classes and GitHub Flow. 💪 

Just a few more adjustments to improve and complete the project. Please find them listed below.
### Status: Changes Required ♻️ 

- [x] [REQUIRED] The Pull Request should have a meaningful title. Something like in the image below.
Screen Shot 2020-07-09 at 4 37 08 PM

- [x] [REQUIRED] The search icon is not positioned correctly in the search input. This is what your page shows:
Screen Shot 2020-08-17 at 5 04 52 PM

- [x] [REQUIRED] The `Up Next` title and `Autoplay` icons are missing in your side bar.
This is what is present in the original:
Screen Shot 2020-08-17 at 5 10 39 PM 

- [x] [REQUIRED ] The footer is missing from your page. It should look like this:
Screen Shot 2020-08-17 at 5 13 37 PM

- [x] [REQUIRED]  Your project should match the appearance of [this archived youtube video player page. ](http://archive.vn/Bss88)

**NB:** The web page was reviewed at a screen size of **1440px.** Kindly state the screen size the project was built at if the page displays better at that size.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

**Comment by GzimAsani** [comment]
> ## Awesome work @GzimAsani and @Rayan84 
> Your project displays good use of CSS classes and GitHub Flow. 
> 
> Just a few more adjustments to improve and complete the project. Please find them listed below.
> 
> ### Status: Changes Required 
> * [x]  [REQUIRED] The Pull Request should have a meaningful title. Something like in the image below.
>   Screen Shot 2020-07-09 at 4 37 08 PM
> * [x]  [REQUIRED] The search icon is not positioned correctly in the search input. This is what your page shows:
>   Screen Shot 2020-08-17 at 5 04 52 PM
> * [x]  [REQUIRED] The `Up Next` title and `Autoplay` icons are missing in your side bar.
>   This is what is present in the original:
> 
> Screen Shot 2020-08-17 at 5 10 39 PM
> 
> * [x]  [REQUIRED ] The footer is missing from your page. It should look like this:
> 
> Screen Shot 2020-08-17 at 5 13 37 PM
> 
> * [x]  [REQUIRED]  Your project should match the appearance of [this archived youtube video player page. ](http://archive.vn/Bss88)
> 
> **NB:** The web page was reviewed at a screen size of **1440px.** Kindly state the screen size the project was built at if the page displays better at that size.
> 
> Happy Coding  
> 
> ###### You can contact me on slack (Chris Amanor) for any clarification.

Hello Christopher, thank you so much for you detailed review. We appreciate your time. We will check our errors and after fixing them we would request another review.

**Comment by GzimAsani** [comment]
Kindly reminding that the project was build in 1366px.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,381 @@
...
+    height: 30px;
+  }
+  .location{
+    width: 15%;
+    height: 30px;
+  }
+  .restricted-mode{
+    width: 15%;
+    height: 30px;
+  }
```
Comment (andresporras3423) [inline style.css]: Group selectors with the same style definition

File: style.css [hunk 7]
```diff
@@ -0,0 +1,381 @@
...
+  .history{
+    width: 8%;
+    height: 30px;
+  }
+  .help {
+    width: 8%;
+    height: 30px;
+  }
```
Comment (andresporras3423) [inline style.css]: Group selectors

File: index.html [hunk 8]
```diff
@@ -1,11 +1,447 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <script src="https://kit.fontawesome.com/5b1eeceadd.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="style.css">
```
Comment (andresporras3423) [inline index.html]: remove repeated link

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @GzimAsani @Rayan84** 
Good job with your new submission for this project :muscle:  :muscle:
Your page meets most of the requirements :clap: :clap: :clap:  
There are just a few details to improve before you can go ahead:
- [x] In the HTML file, remove the repeated link to the style.css file
- [x] Don't repeat yourself, group selectors with the same style definition: https://www.w3schools.in/css3/group-selector/

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Review by MuriloRoque — APPROVED** [review]
# Status: Approved :green_circle: 

## Hello @GzimAsani and @Rayan84 :wave: 

Thank you for making all of the required changes from the last TSE, your project is approved, congratulations :clap: :clap: :clap: 

You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**